libs/SalesforceNetwork/SalesforceNetwork.xcodeproj/xcshareddata/xcschemes/SalesforceNetworkiOS.xcscheme

```diff
@@ -0,0 +1,169 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Scheme
+   LastUpgradeVersion = "0640"
+   version = "1.3">
```

Comment:
Why have we renamed this scheme to ```SalesforceNetworkiOS```? Could we revert this to ```SalesforceNetwork``` as it was before please?